examples/AWSDriverExample/src/main/java/software/amazon/TelemetryTracingXRayExample.java

```diff
@@ -0,0 +1,101 @@
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon;
+
+import com.amazonaws.xray.AWSXRay;
+import com.amazonaws.xray.AWSXRayRecorderBuilder;
+import com.amazonaws.xray.entities.Segment;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import software.amazon.jdbc.PropertyDefinition;
+
+public class TelemetryTracingXRayExample {
+
+  // User configures connection properties here
+  public static final String POSTGRESQL_CONNECTION_STRING =
+      "jdbc:aws-wrapper:postgresql://atlas-postgres.cluster-czygpppufgy4.us-east-2.rds.amazonaws.com:5432/postgres";
+
+  private static final String USERNAME = "username";
+  private static final String PASSWORD = "password";
+  private static final String SQL_DBLIST = "select datname from pg_database;";
+  private static final String SQL_SLEEP = "select pg_sleep(20);";
+  private static final String SQL_TABLELIST = "select * from information_schema.tables where table_schema='public';";
+
+  public TelemetryTracingXRayExample() {
+  }
+
+  public void runExampleQuery1(Properties properties) throws SQLException {
+    try (final Connection conn = DriverManager.getConnection(POSTGRESQL_CONNECTION_STRING, properties);
+         final Statement statement = conn.createStatement();
+         final ResultSet rs = statement.executeQuery(SQL_DBLIST)) {
+      //AWSXRay.beginSegment("part of application");
+      System.out.println(Util.getResult(rs));
+      //AWSXRay.endSegment();
+    }
+  }
+
+  public void runExampleQuery2(Properties properties) throws SQLException {
+    try (final Connection conn = DriverManager.getConnection(POSTGRESQL_CONNECTION_STRING, properties);
+         final Statement statement = conn.createStatement();
+         final ResultSet rs = statement.executeQuery(SQL_TABLELIST)) {
+      //AWSXRay.beginSegment("part of application");
+      System.out.println(Util.getResult(rs));
+      //AWSXRay.endSegment();
+    }
+  }
+
+  public void runExampleQuery3(Properties properties) throws SQLException {
+    try (final Connection conn = DriverManager.getConnection(POSTGRESQL_CONNECTION_STRING, properties);
+         final Statement statement = conn.createStatement();
+         final ResultSet rs = statement.executeQuery(SQL_SLEEP)) {
+      //AWSXRay.beginSegment("part of application");
+      System.out.println(Util.getResult(rs));
+      //AWSXRay.endSegment();
+    }
+  }
+
+  public static void main(String[] args) throws SQLException {
+    final TelemetryTracingXRayExample example = new TelemetryTracingXRayExample();
```

Comment:
Instantiating an object for a simple demo class is a bit unusual. Can we just make runExampleQueryX() static and use them directly in main()?